src/fosslight_util/parsing_yaml.py

```diff
@@ -7,11 +7,14 @@
 import logging
 import codecs
 import os
+import re
 import sys
 from .constant import LOGGER_NAME
 from .oss_item import OssItem
 
 _logger = logging.getLogger(LOGGER_NAME)
+SUPPORT_OSS_INFO_FILES = ["oss-pkg-info.yaml", "sbom-info.yaml"]
```

Comment:
`"oss-pkg-info[\s\S]*.yaml",  r"sbom-info[\s\S]*.yaml"`
Shouldn't it be like this?